### ISSUE TITLE
v5.5 Testing

### ISSUE THREAD

**Issue by phpgraham** [rappasoft/laravel-5-boilerplate#979]
Clean install no changes yet, other than creating Model with:
php artisan make:model Pages -mr
created a route and view, which works fine in browser
then created a simple test, using php artisan make:test

namespace Tests\Feature;
use Tests\TestCase;
use Illuminate\Foundation\Testing\RefreshDatabase;

class PagesTest extends TestCase
{
    /**
     * A basic test example.
     *
     * @return void
     */
    public function testExample()
    {
        $this->assertTrue(true);
    }
}

Get error:
Fatal error: Class 'Tests\TestCase' not found in C:\Homestead_Projects\laracmsnew\tests\Feature\PagesTest.php on line 8
PHP Fatal error:  Class 'Tests\TestCase' not found in C:\Homestead_Projects\laracmsnew\tests\Feature\PagesTest.php on line 8

Line 8 is:  class PagesTest extends TestCase

This simple test works fine on a clean laravel 5.5 install on same server with same install procedures.

Is Tests\TestCase extends Illuminate\Foundation\Testing\TestCase  working on BoilerPlate  or is it just BrowserKitTesting that works, the 92 tests that come with BoilerPlate all work

How can i run phpunit tests ?

**Comment by SketchNI** [comment]
Hi. I followed your steps and got the same error your did

`PHP Fatal error:  Class 'Tests\TestCase' not found in /home/sketch/Code/L5B/tests/Feature/PageTest.php on line 6`

I looked in `TestCase.php` and it doesn't have a namespace so I removed that declaration and still had the same issue. I looked at some of the other tests and saw they were using BrowserKitTesting and changing `class PageTest extends TestCase` to `class PageTest extends BrowserKitTestCase` allows the tests to run successfully.

My PagesController@index just returns "OK" to the browser so writing this:
````php
<?php namespace Tests\Feature;

use Illuminate\Foundation\Testing\RefreshDatabase;
use Tests\BrowserKitTestCase;

class PageTest extends BrowserKitTestCase
{
    /**
     * A basic test example.
     *
     * @return void
     */
    public function testExample()
    {
        $this->visit('/pages')
            ->see("OK")
            ->assertTrue(true);
    }
}
````
allows me to confirm that BrowserKitTestCase works.
